### ISSUE TITLE
`max_execution_time` unavailable for Mysql 5.6+

### ISSUE THREAD

**Issue by brandonjmckay** [ankane/strong_migrations#157]
I'm on mysql 5.6, and when we try to install this gem and run a migration we get the following error:
```
Mysql2::Error: Unknown system variable 'max_execution_time'
```

This is coming from [this line in `StrongMigrations::Checker`](https://github.com/ankane/strong_migrations/blob/d4bf71467a/lib/strong_migrations/checker.rb#L359). It seems like this attribute is no longer available in newer version of mysql

**Comment by ankane** [comment]
`max_execution_time` isn't available in MySQL 5.6, so you'll need to comment out the `statement_timeout` line in the initializer.

**Comment by brandonjmckay** [comment]
thanks so much!
